lib/driver.js

```diff
@@ -1079,11 +1079,6 @@ class XCUITestDriver extends BaseDriver {
     if (this.isSafari()) {
       return;
     }
-    // if user has passed in desiredCaps.autoLaunch = false
-    // meaning they will manage app install / launching
-    if (this.opts.autoLaunch === false) {
```

Comment:
how does removing this code change the behavior to be correct?

Reply:
the name of the capability is autoLaunch, which means it should have nothing to do with app installation. Also, if one wants to avoid an app to be installed, then he could always only provide a bundle Id instead of an application path

Reply:
ah ok then my suggestion would need to be reworded to not mention installation i guess.